test/encrypt.c

```diff
@@ -425,8 +425,7 @@ int _ValidateEncrypt(const cn_cbor * pControl, const byte * pbEncoded, size_t cb
 		else if ((pFail != NULL) && (pFail->type != CN_CBOR_TRUE)) fFail = true;
 
                 size_t cb;
-                byte * pb;
-                pb = COSE_Encrypt_GetContent(hEnc, &cb, NULL);
```

Comment:
This assignment is for debugging purposes.  If there is a failure then it is easier to assign to a variable.  Otherwise the call itself does not really make any sense.

Reply:
ok, if only for debugging then we can put a `#ifdef DEBUG` around this?

Reply:
Yes, but since this is just the test program why would I care?